### PR TITLE
upstream v5.52.0

### PR DESCRIPTION
**Pull request by flostadler**
- **Moving ./upstream to v5.52.0**
- **Update patches**

### CODE REVIEW

**Comment by iwahbe** [comment]
Looks like you are missing a `make tfgen build_sdks`

**Comment by flostadler** [comment]
> Looks like you are missing a `make tfgen build_sdks`

Yeah missed pushing that commit :(

File: sdk/nodejs/kinesisanalyticsv2/application.ts [hunk 1]
```diff
@@ -262,6 +262,10 @@ export class Application extends pulumi.CustomResource {
      * The application's configuration
      */
     public readonly applicationConfiguration!: pulumi.Output<outputs.kinesisanalyticsv2.ApplicationApplicationConfiguration>;
+    /**
+     * The application's mode. Valid values are `STREAMING`, `INTERACTIVE`.
```
Comment (t0yv0) [inline sdk/nodejs/kinesisanalyticsv2/application.ts:266]: Pity not an enum here.

**Comment by t0yv0** [comment]
https://github.com/pulumi/schema-tools/issues/73

**Review by t0yv0 — APPROVED** [review]

**Pull request merged**